solid/src/main/java/com/inrupt/client/solid/SolidClient.java

```diff
@@ -64,11 +57,22 @@ public class SolidClient {
     private final Client client;
     private final Headers defaultHeaders;
     private final boolean fetchAfterWrite;
+    private final JsonService jsonService;
 
     SolidClient(final Client client, final Headers headers, final boolean fetchAfterWrite) {
         this.client = Objects.requireNonNull(client, "Client may not be null!");
         this.defaultHeaders = Objects.requireNonNull(headers, "Headers may not be null!");
         this.fetchAfterWrite = fetchAfterWrite;
+
+        // It is acceptable for a SolidClient instance to be in a classpath without any implementation for
+        // JsonService, in which case the ProblemDetails exceptions will fallback to default and not be parsed.
+        JsonService js;
```

Comment:
Local is redundant.

Reply:
This has now been removed. I had a compilation error because of this, which in retrospective was due to the member variable being immutable and set both in the `try` and `catch` blocks, which I thought should be okay as I wouldn't expect both assignments to be able to happen consecutively.